daffodil-test-ibm1/src/test/resources/test-suite/tresys-contributed/sepSuppression.tdml

```diff
@@ -58,10 +58,10 @@
       for exploring when groups are 'potentially trailing' meaning their 
       separator (in enclosing sequence) might be suppressed/not-required.
      -->
-    <xs:element name="ptg1" dfdl:initiator="%NL;">
+    <xs:element name="ptg1" dfdl:initiator="|">
```

Comment:
Also not expecting these changes in this change set. These have nothing to do with the continue-on-error from github actions? Or are these the tests that were failing but were being masked?